tldw_Server_API/tests/Chat_NEW/unit/test_queue_status_endpoint.py

```
@@ -17,22 +20,132 @@ def get_queue_status(self):
 
+    def get_recent_activity(self, limit: int = 50):
+        return [{"id": 1, "status": "ok", "limit": limit}]
+
+
+async def _principal_override(request=None):  # type: ignore[override]
+    principal = AuthPrincipal(
+        kind="user",
+        user_id=1,
+        api_key_id=None,
+        subject="admin",
+        token_type="access",
```

Ruff (0.14.6)

Possible hardcoded password assigned to argument: "token_type"

(S106)